hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/client/HddsClientUtils.java

```diff
@@ -68,10 +73,26 @@ private static void doNameChecks(String resName, String resType) {
       throw new IllegalArgumentException(resType + " name is null");
     }
 
-    if (resName.length() < OzoneConsts.OZONE_MIN_BUCKET_NAME_LENGTH ||
-        resName.length() > OzoneConsts.OZONE_MAX_BUCKET_NAME_LENGTH) {
+    if (resName.length() < OzoneConsts.OZONE_MIN_BUCKET_NAME_LENGTH) {
       throw new IllegalArgumentException(resType +
-          " length is illegal, " + "valid length is 3-63 characters");
+          " name '" + resName + "' is too short, " +
+          "valid length is 3-63 characters");
```

Comment:
Let's create a String constant for `"valid length is 3-63 characters"`, using actual `OZONE_MIN_BUCKET_NAME_LENGTH` and `OZONE_MAX_BUCKET_NAME_LENGTH`, and reuse it for both checks.

Reply:
Good point, thank you! Added in the latest commit. That also made `doTestBadResourceNameReported()` specific to length-related assertions, so I adjusted its name accordingly.